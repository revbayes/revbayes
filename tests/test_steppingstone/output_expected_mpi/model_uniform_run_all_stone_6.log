Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	gamma_rates[1]	gamma_rates[2]	gamma_rates[3]	gamma_rates[4]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-23248.74	-23239.98	-8.766277	0.9613009	0.0866421	0.126256	0.095946	0.101924	0.0385189	0.167482	0.432175	0.148232	0.128103	0.157482	0.198599	0.203314	0.0693174	0.0471496	0.107529	0.122145	0.279697	0.0934155	0.222573	0.225615	0.068821	0.133909	0.453271	0.310031	0.0201625	0.00348758	0.0784379	0.120691	0.0325115	0.194581	0.0627655	0.136288	0.138343	0.00189904	0.0422505	0.0509846	0.08184	0.204831	0.0067797	0.0201613	0.0867395	0.164122	0.0745582	0.0104983	0.343827	0.0927844	0.0176205	0.50943	0.0258401	0.129169	0.464803	0.99328	2.41275	0.274645	0.355733	0.118262	0.25136	0.6932527	5.539584
10	-23466.19	-23467.28	1.091038	0.3797113	0.0717822	0.08585	0.0994659	0.0461799	0.0276857	0.142656	0.16743	0.0473316	0.215341	0.0967909	0.217528	0.144918	0.042081	0.158807	0.140008	0.090087	0.141722	0.100421	0.153667	0.0854498	0.0444892	0.0873168	0.33868	0.149729	0.0854716	0.00851398	0.0148567	0.0779976	0.0372224	0.0502314	0.0496689	0.0330062	0.356251	0.00590047	0.0490335	0.0954046	0.0397227	0.272475	0.00722451	0.0372334	0.135403	0.0548999	0.0185557	0.0462519	0.223014	0.199419	0.017537	0.485714	0.0280642	0.0138996	0.166445	0.708358	3.1113	0.243098	0.392506	0.131163	0.233233	0.2008419	4.324488
