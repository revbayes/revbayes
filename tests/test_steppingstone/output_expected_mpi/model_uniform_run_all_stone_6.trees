Iteration	Posterior	Likelihood	Prior	psi
0	-23248.74	-23239.98	-8.766277	(((((Colobus_guereza[&index=6]:0.167482,(Hylobates_lar[&index=9]:0.128103,Pan_paniscus[&index=17]:0.279697)[&index=32]:0.136288)[&index=25]:0.020162,((Aotus_trivirgatus[&index=1]:0.086642,(Saimiri_sciureus[&index=20]:0.225615,(Callicebus_donacophilus[&index=2]:0.126256,Cebus_albifrons[&index=3]:0.095946)[&index=33]:0.138343)[&index=26]:0.003488)[&index=43]:0.074558,Macaca_mulatta[&index=13]:0.069317)[&index=42]:0.164122)[&index=30]:0.194581,Chlorocebus_aethiops[&index=5]:0.038519)[&index=31]:0.062766,Daubentonia_madagascariensis[&index=7]:0.432175)[&index=34]:0.001899,Galeopterus_variegatus[&index=23]:0.453271,(((Loris_tardigradus[&index=12]:0.203314,(Perodicticus_potto[&index=18]:0.093415,Galago_senegalensis[&index=8]:0.148232)[&index=41]:0.086740)[&index=28]:0.120691,Nycticebus_coucang[&index=15]:0.107529)[&index=29]:0.032511,((Tarsius_syrichta[&index=21]:0.068821,Otolemur_crassicaudatus[&index=16]:0.122145)[&index=37]:0.081840,((Lemur_catta[&index=10]:0.157482,(Lepilemur_hubbardorum[&index=11]:0.198599,((Microcebus_murinus[&index=14]:0.047150,Cheirogaleus_major[&index=4]:0.101924)[&index=40]:0.020161,Propithecus_coquereli[&index=19]:0.222573)[&index=38]:0.204831)[&index=35]:0.042251)[&index=39]:0.006780,Varecia_variegata_variegata[&index=22]:0.133909)[&index=27]:0.078438)[&index=36]:0.050985)[&index=24]:0.310031)[&index=44];
10	-23466.19	-23467.28	1.091038	((((Loris_tardigradus[&index=12]:0.144918,Perodicticus_potto[&index=18]:0.100421)[&index=41]:0.135403,(((Nycticebus_coucang[&index=15]:0.140008,((Tarsius_syrichta[&index=21]:0.044489,(Daubentonia_madagascariensis[&index=7]:0.167430,((Macaca_mulatta[&index=13]:0.042081,(((Aotus_trivirgatus[&index=1]:0.071782,Callicebus_donacophilus[&index=2]:0.085850)[&index=43]:0.018556,Cebus_albifrons[&index=3]:0.099466)[&index=33]:0.356251,Saimiri_sciureus[&index=20]:0.085450)[&index=26]:0.008514)[&index=36]:0.095405,((Pan_paniscus[&index=17]:0.141722,(Hylobates_lar[&index=9]:0.215341,Colobus_guereza[&index=6]:0.142656)[&index=32]:0.033006)[&index=25]:0.085472,Chlorocebus_aethiops[&index=5]:0.027686)[&index=31]:0.049669)[&index=42]:0.054900)[&index=34]:0.005900)[&index=30]:0.050231,Varecia_variegata_variegata[&index=22]:0.087317)[&index=37]:0.039723)[&index=29]:0.037222,Lemur_catta[&index=10]:0.096791)[&index=27]:0.014857,(Lepilemur_hubbardorum[&index=11]:0.217528,(Microcebus_murinus[&index=14]:0.158807,(Cheirogaleus_major[&index=4]:0.046180,Propithecus_coquereli[&index=19]:0.153667)[&index=40]:0.037233)[&index=38]:0.272475)[&index=35]:0.049034)[&index=39]:0.007225)[&index=28]:0.077998,Galago_senegalensis[&index=8]:0.047332)[&index=24]:0.149729,Galeopterus_variegatus[&index=23]:0.338680,Otolemur_crassicaudatus[&index=16]:0.090087)[&index=44];
