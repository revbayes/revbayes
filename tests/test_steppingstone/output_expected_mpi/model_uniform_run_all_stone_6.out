state	power	likelihood
4	0.01024	-23783.4
5	0.01024	-23829.4
6	0.01024	-24056.4
7	0.01024	-23839
8	0.01024	-23909.3
9	0.01024	-23680.3
10	0.01024	-23467.3
